tests/test_pip.py

```diff
@@ -0,0 +1,43 @@
+import pytest
+
+from testinfra.utils.ansible_runner import AnsibleRunner
+
+
+@pytest.fixture()
+def AnsibleDefaults(Ansible):
+    """ Load default variables into dictionary.
+
+    Args:
+        Ansible - Requires the ansible connection backend.
+    """
+    return Ansible("include_vars", "./defaults/main.yml")["ansible_facts"]
+
+
+@pytest.fixture()
+def AnsibleFacts(Ansible):
+    """ Load ansible facts into a dictionary.
+
+    Args:
+        Ansible - Requires ansible connection backend.
+    """
+
+    return Ansible("setup")["ansible_facts"]
+
+
+def test_pip_exists(Package, AnsibleDefaults, AnsibleFacts):
+    """ Ensure pip is installed, with the proper version for the OS.
+
+    Args:
+        Package - Module to determine package install status and version
+        AnsibleDefaults - Module to determine pip install version
+        AnsibleFacts - Used to determine OS Distribution Version
+    """
+    pip = Package("python-pip")
+    os_version = AnsibleFacts["ansible_distribution_version"]
+    if float(os_version) >= 16.04:
```

Comment:
We can leverage Molecule's `host_vars` configuration option for a workaround. If `host_vars` are set in `molecule.yml`, it writes a `host_vars` file for Ansible to use during provisioning.  I can then load that file into testinfra for testing. That way we only have to specify the target `pip_version` in `molecule.yml` one time, rather than having to maintain it throughout the playbook and tests. 

Using this fix in [`1226538`](https://github.com/azavea/ansible-pip/pull/2/commits/1226538748bb90b4dbcd30c823189cfcd3b71a3e)